test/tests/nek_errors/deformation/nek2.i

```diff
@@ -0,0 +1,105 @@
+[Mesh]
+  type = NekRSMesh
+  order = SECOND
+  volume = true
+  parallel_type = replicated
+  displacements = 'disp_x disp_y disp_z'
+#  use_displaced_mesh = true
+[]
+
+[Problem]
+  type = NekRSProblem
+  casename = 'nekbox'
+  moving_mesh = true
+[]
+
+[AuxVariables]
+  [dummy]
+  []
+  [disp_x]
+    order = SECOND
+  []
+  [disp_y]
+    order = SECOND
+  []
+  [disp_z]
+    order = SECOND
+  []
+[]
+
+# This AuxVariable and AuxKernel is only here to get the postprocessors
+# to evaluate correctly. This can be deleted after MOOSE issue #17534 is fixed.
+[AuxKernels]
+  [dummy]
+    type = ConstantAux
+    variable = dummy
+    value = 0.0
+  []
+[]
+
+[Executioner]
+  type = Transient
+  [TimeStepper]
+    type = NekTimeStepper
+  []
+  [Quadrature]
+    type = GAUSS_LOBATTO
+    order = SECOND
+  []
+[]
+
+[Postprocessors]
```

Comment:
For these two tests, let's delete all the postprocessors, source terms, and other extraneous stuff in these input files. If we need to change any syntax in the future, reducing things not actually used by the test will help.

Reply:
Thanks, I think I have removed everything. Tests are still working fine.